org.gnome.Boxes.json

```diff
@@ -0,0 +1,323 @@
+                }
+            ]
+        },
+        {
+            "name" : "gtk-vnc",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://git.gnome.org//browse/gtk-vnc",
+                    "tag" : "v0.7.2",
+                    "commit" : "0e8b3c545ad109051501b4ff4ed548689df6e7d4"
+                }
+            ]
+        },
+        {
+            "name" : "libnl",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://www.infradead.org/~tgr/libnl/files/libnl-1.1.4.tar.gz",
+                    "sha256" : "4f80c21fe5bbcdde6e72b59b4f98306063a41421f909887c34e58d93e746d063"
+                }
+            ]
+        },
+        {
+            "name" : "libusb",
+            "config-opts" : [
+                "--disable-udev"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/libusb/libusb/releases/download/v1.0.22/libusb-1.0.22.tar.bz2",
+                    "sha256" : "75aeb9d59a4fdb800d329a545c2e6799f732362193b465ea198f2aa275518157"
+                }
+            ]
+        },
+        {
+            "name" : "yajl",
+            "buildsystem" : "cmake",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/lloyd/yajl/archive/2.1.0.tar.gz",
+                    "sha256": "3fb73364a5a30efe615046d07e6db9d09fd2b41c763c5f7d3bfb121cd5c5ac5a"
+                }
+            ]
+        },
+        {
+            "name" : "libvirt",
+            "config-opts" : [
+                "--without-html-subdir",
+                "--without-storage-mpath"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://libvirt.org/sources/libvirt-4.1.0.tar.xz",
+                    "sha256" : "8a2fa4826f311a936be8b7d4c8d76516c29417a593b1d1bb8641a8caaa316439"
+                }
+            ]
+        },
+        {
+            "name" : "libvirt-glib",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "http://libvirt.org/sources/glib/libvirt-glib-1.0.0.tar.gz",
+                    "sha256" : "ca9158d01cc4d9e0218f94f8c9ee04ae367d8c3b4396daaebca2c4b89a2e8a47"
+                }
+            ]
+        },
+        {
+            "name" : "spice-protocol",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://www.spice-space.org/download/releases/spice-protocol-0.12.13.tar.bz2",
+                    "sha256" : "89ee11b202d2268e061788e6ace114e1ff18c7620ae64d1ca3aba252ee7c9933"
+                }
+            ]
+        },
+        {
+            "name" : "celt051",
```

Comment:
this needs to be reconfigured to work with aarch64:

https://flathub.org/builds/#/builders/36/builds/161

Example of how to do that here: https://github.com/flathub/shared-modules/blob/08e9b9eead7a95139be5d1fbc24bea730fb78f7a/SDL/SDL_net-1.2.8.json